source/rust_verify/tests/common/mod.rs

```diff
@@ -137,10 +139,16 @@ pub fn verify_files_and_pervasive(
     let files = files.into_iter().map(|(p, f)| (p.into(), f)).collect();
     let captured_output = std::sync::Arc::new(std::sync::Mutex::new(Vec::new()));
     let captured_output_1 = captured_output.clone();
+
+    let pervasive_path = match std::env::var("PERVASIVE_PATH") {
```

Comment:
Can you rename this to TEST_PERVASIVE_PATH to distinguish it from `--pervasive-path`?